manifests/transport.pp

```diff
@@ -49,7 +49,7 @@
 ) {
   include ::postfix::augeas
 
-  $smtp_nexthop = ("${nexthop}" =~ /\[.*\]/)
+  $smtp_nexthop = ($nexthop =~ /\[.*\]/)
```

Comment:
This won't work if `$nexthop` is `undef`:

```
Error: Evaluation Error: Left match operand must result in a String value. Got an Undef Value.
```

Reply:
Yeah, overlook that one, my bad. Now casting explicitly to `String` via the data type.